examples/views/wait_for_button_press.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+
+import discord
+from discord.ext import commands
+from discord.ui import Button, View
+
+
+bot = commands.Bot(command_prefix="!")
+
+# Create a slash command for the supplied guilds.
+@bot.slash_command(name="button", guild_ids=[...], description="Wait for button press example from pycord") 
```

Comment:
Why did you set the `name` parameter here? The function has the same name as what's set in the `name` parameter, so it's pointless

Reply:
There are issues with registering commands that have duplicate function names, so setting `name` is a best practice to ensure that your command names are explicitly defined.

Reply:
> (not directly related to this PR)
Then it doesn't matter. This is just an example, after all

Reply:
We should be promoting best practices everywhere, including in examples.

Reply:
Alright fine